sources/Makefile.am

```diff
@@ -13,9 +13,9 @@ sources =
 
 build_sources = AssemblyInfo.cs $(sources)
 
-CLEANFILES = $(DLL) generated-stamp generated/*.cs $(API)
+CLEANFILES = $(DLL) generated-stamp generated/* $(API)
 
-DISTCLEANFILES = AssemblyInfo.cs $(DLLMAP)
+DISTCLEANFILES = AssemblyInfo.cs
```

Comment:
Sure, that one probably was a mistake. I'm preparing a new patch series for webkitgtk-sharp and soup-sharp. Stay tuned ;)